File: admin_ui/report_admin_ui/src/app/features/auth/reports-controller/models/dto-meta-data-response.ts
import { DtoReportImage } from "./dto-report-image";

export class DtoMetaDataResponse {
    reportUrl?: string;
    reportHeading?: string;
    reportDesc?: string;
    keyWords?: string;
    pageTitle?: string;
    excelSaveFileName?:string;
    images?: DtoReportImage[];

}

File: back_end/MindEarth/MindEarth.Web/Features/Reports/DTO/DTO_MetaDataResponse.cs
using DocumentFormat.OpenXml.ExtendedProperties;

namespace MindEarth.Web.Features.Reports.DTO
{
    public class DTO_MetaDataResponse
    {
        public string ReportUrl { get; set; }
        public string ReportHeading { get; set; }
        public string ReportDesc { get; set; }
        public string KeyWords { get; set; }
        public string PageTitle { get; set; }

        public string ExcelSaveFileName { get; set; }
        public List<DTO_ReportImage> Images  { get; set; }

    }
}
